Wikipedia/Code/TalkPageTopicListViewController.swift

```diff
@@ -68,9 +68,13 @@ class TalkPageTopicListViewController: ColumnarCollectionViewController {
         completedActivityType = nil
     }
     
-    override func traitCollectionDidChange(_ previousTraitCollection: UITraitCollection?) {
-        super.traitCollectionDidChange(previousTraitCollection)
-        cellLayoutEstimate = nil
```

Comment:
was it intentional to remove `cellLayoutEstimate = nil`?

Reply:
no, thanks for catching!